lib/chef/resource/aws_route_table.rb

```diff
@@ -35,6 +35,16 @@ class Chef::Resource::AwsRouteTable < Chef::Provisioning::AWSDriver::AWSResource
   attribute :vpc,    kind_of: [ String, AwsVpc, AWS::EC2::VPC ], required: true
 
   #
+  # Enable route propagation from one or more virtual private gateways
+  #
+  # The value should be an array of virtual private gateway ID:
+  # ```ruby
+  # virtual_private_gateways ['vgw-abcd1234', 'vgw-abcd5678']
+  # ```
+  #
+  attribute :virtual_private_gateways, kind_of: [ String, Array ]
```

Comment:
To make sure this is an array when in the provider you can add
```ruby
attribute :virtual_private_gateways, kind_of: [ String, Array ], coerce: proc { |v| [v].flatten }
```

`coerce` is defined at https://github.com/chef/chef-provisioning-aws/blob/master/lib/chef/provisioning/aws_driver/super_lwrp.rb#L15 and is only supported for the AWS resources (currently).  But its super handy!